opennms-base-assembly/src/main/filtered/etc/jmx-datacollection-config.d/kafka.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<jmx-datacollection-config rrdRepository="${install.share.dir}/rrd/snmp/">
+    <jmx-collection name="jmx-kafka">
+        <rrd step="300">
+            <rra>RRA:AVERAGE:0.5:1:2016</rra>
+            <rra>RRA:AVERAGE:0.5:12:1488</rra>
+            <rra>RRA:AVERAGE:0.5:288:366</rra>
+            <rra>RRA:MAX:0.5:288:366</rra>
+            <rra>RRA:MIN:0.5:288:366</rra>
+        </rrd>
+        <mbeans>
+            <mbean name="JVM Memory" objectname="java.lang:type=OperatingSystem">
```

Comment:
You could also use the '<import-mbeans>' tag to avoid having to redefine these in every collection. See https://github.com/OpenNMS/opennms/blob/develop/opennms-base-assembly/src/main/filtered/etc/examples/jvm-datacollection/jmx-datacollection-config.d/activemq.xml#L11 for an example.